### ISSUE TITLE
Feature Request: ability to block brim/brim ears

### ISSUE THREAD

**Issue by jhoblitt** [supermerill/SuperSlicer#797]
The brim ears feature is placing ears on an internal channel of a Prusa extruder part where they would be difficult to remove.

![Screenshot from 2020-12-12 09-05-03](https://user-images.githubusercontent.com/948901/101988783-29719e80-3c59-11eb-95a5-be8260a32093.png)

It would be useful if there was an equivalent to support blockers for brims/brim ears such that I could block the brim on all but the "outermost" perimeter.

[x-carriage.zip](https://github.com/supermerill/SuperSlicer/files/5683357/x-carriage.zip)

**Comment by jpGuppy** [comment]
I also agree that a modifier box for brim would be useful, for cases where you may want ears for sharp corners, but not for close jointed areas like in this articulating pangolin. 

![Pangolin_bottom](https://user-images.githubusercontent.com/97979213/152827507-60a358e2-682f-42bb-86a6-0b3411befebc.PNG)
